Added Figure to Results and Discussion

fsl506 opened the pull request:
Writing results and discussion in this branch then shall merge once approved.

fsl506 committed:
Added Figure to Results and Discussion

fsl506 committed:
Added Simulation Results Figure

fsl506 committed:
Increasing image size

fsl506 committed:
Further image size increase

fsl506 committed:
Typo Ti-6Al-5V Corrected

fsl506 committed:
Temp/displacement discussion added

fsl506 committed:
Stress distribution results added

mtp19stl reviewed (approved):
Looks good! Just wondering whether we would expect highest stress at the highest temperature or if it would be easier to deform at higher temperature?

fsl506 commented:
Yeah, I'm really not sure how to explain the results we've gotten... Because it's not what you'd expect... How would you write it? Feel free to edit

fsl506 committed:
Gotten rid of expectations

fsl506 commented:
I've gotten rid of 'as expected.'... I don't know what else to add instead so I think leaving it needing more research is best. Thoughts?

Sakina-1 reviewed (approved):
This looks good to me. You're right in saying the temp distribution is similar and the x and y displacement is small in comparison to some test cases. This is because, as you said, the thermal conductivities/expansions are similar for beta and alpha. The purpose of the test cases was to exaggerate the values for thermal conductivities/expansions just to check is everything is working. In terms of the stress plot, I'm not sure why the stress is consistent in the alpha grains, I'm happy to rerun the simulation to see if I get anything different?

fsl506 committed:
Wrong stress graph

fsl506 committed:
Corrected Final Simulation Results

fsl506 committed:
Deleting incorrect stress discussion

fsl506 committed:
Added limitations paragraph

fsl506 committed:
Added reference Ti64 Phase Diagram

fsl506 committed:
Added discussion of stress distribution results

fsl506 commented:
Updated the stress distribution discussion and added a few sentences on the model limitations. Let me know if it makes sense (particularly from a materials POV)

fsl506 committed:
Deleted reference to prevent conflict

Reference will be added to master branch instead

fsl506 committed:
Merge branch 'master' into fsl506-ResultsandDiscussion

fsl506 committed:
Merge branch 'master' into fsl506-ResultsandDiscussion

mtp19stl reviewed (changes requested):
Looks good Frances! The new stress plot looks much better and the results seem correct to me.

One small suggestion, I would maybe say that there are stress concentrations at the grain boundaries in the second paragraph rather than stress builds.

mtp19stl commented:
@fsl506 Just had another thought; we could mention that the stress being higher in alpha is accurate as alpha is less ductile than beta due to being hcp, whereas beta is bcc and has lots of slip planes

fsl506 commented:
@mtp19stl Yeah, so this is what I was confused about because it's not measuring shear stress, it's about thermal stress... So does that still apply then? What I researched about thermal stress suggested that it was to do with the CTE but I have no idea.

fsl506 committed:
Updated Sam's changes

mtp19stl commented:
@fsl506 I'm not 100%, but i think the thermal expansion/contraction causes deformation which causes stress due to the resistance to the deformation. And because alpha is less ductile due to it being hcp it has greater resistance to the deformation and therefore greater stresses. 

What do you think @mjki4jcd ?

JoshUoM reviewed (approved):
Looks good!

mtp19stl reviewed (approved):
Looks good to me

fsl506 committed:
Updated corrected stresses and minor changes

Sakina-1 reviewed (approved):
Looks good to me :)

Gbowker reviewed (commented):
We discussed this over a zoom meeting 19/06/20. Looks good. Just finishing touches now.
